sunpy/coordinates/transformations.py

```diff
@@ -66,6 +68,28 @@
            'hme_to_gei', 'gei_to_hme', 'gei_to_gei']
 
 
+# Boolean flag for whether to ignore the inertial motion of the Sun
+_ignore_sun_motion = False
+
+
+@contextmanager
+def ignore_sun_motion():
+    """
+    Context manager for ignoring the inertial motion of the Sun for transformations.
```

Comment:
The motion of the Sun presumable isn't quite inertial (because it interacts via. gravity with other bodies)? I would suggest:

```suggestion
    Context manager for ignoring the motion of the center of the Sun for transformations.
```